sru/cuda_functional.py

```diff
@@ -64,6 +64,8 @@ def forward(ctx, u, x, weight_c, bias,
 
         forward_func = sru_cuda_lib.sru_bi_forward if bidirectional else \
             sru_cuda_lib.sru_forward
+        if h.dtype == torch.float16:
+            bias = bias.half()
```

Comment:
One question: why do we need this conversion again given there are `cast()` calls in the `elementwise_recurrence_gpu()` method?

Reply:
I dont know. Everything else arrives as float16. Except bias. For some reason. I guess I can try to dig into why a bit.

Reply:
Hmmm, perhaps I only needed it when I entirely removed the autocast(disabled=True) block. And then I found eerything still worked iwth these two lines in. I think. anyway, I've removed these two lines now.

Reply:
Ok. sounds good to me now.